app/javascript/components/Results/Jobs/Jobs.tsx

```diff
@@ -59,7 +59,7 @@ export const Jobs = ({ jobs=[] }: JobsProps) => {
 
   return (
     <>
-      {jobs?.length > 0 && (
+      {typeof jobs !== 'string' && jobs?.length > 0 && (
```

Comment:
Why do we need to validate that `typeof jobs !== 'string'`? when can `jobs` be a string?